<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>311</width>
      <height>191</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>230</y>
      <width>361</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>161</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_eventname">
       <property name="text">
        <string>Event name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_eventname"/>
     </item>
     <item>
      <widget class="QLabel" name="label_stime">
       <property name="text">
        <string>Start time</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="timeEdit_start"/>
     </item>
     <item>
      <widget class="QLabel" name="label_etime">
       <property name="text">
        <string>End time</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="timeEdit_end"/>
     </item>
     <item>
      <widget class="QPushButton" name="button_calendar">
       <property name="text">
        <string>Confirm</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>American Typewriter</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Admin Mode</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>359</x>
      <y>500</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_clear">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_show_24">
       <property name="text">
        <string>Show events(24)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_show_12">
       <property name="text">
        <string>Show Event(12)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_current_user">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>155</width>
      <height>78</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_clear</sender>
   <signal>clicked()</signal>
   <receiver>textBrowser</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>383</x>
     <y>536</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>506</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
